sos/plugins/openvswitch.py

```diff
@@ -13,10 +13,12 @@
 from os.path import join as path_join
 from os import environ
 
+import re
+
```

Comment:
Just curious: why is this import required here, but it is missing in (otherwise in-par) the plugin in `master` branch?

Reply:
I pushed a fixup commit to `master` after this one that added the import. The plugin does a regex match later one:

https://github.com/sosreport/sos/pull/2248/files#diff-0a9522d996653ba1be2e296998f68ea1R171